packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -579,26 +548,26 @@ export class SummarizerNode implements IRootSummarizerNode {
 		id: string,
 		createParam: CreateChildSummarizerNodeParam,
 	): ICreateChildDetails {
-		let latestSummary: SummaryNode | undefined;
+		let childLastSummaryReferenceSequenceNumber: number | undefined;
 		let changeSequenceNumber: number;
 
-		const parentLatestSummary = this._latestSummary;
+		const parentLastSummaryReferenceSequenceNumber = this._lastSummaryReferenceSequenceNumber;
 		switch (createParam.type) {
 			case CreateSummarizerNodeSource.FromAttach: {
 				if (
-					parentLatestSummary !== undefined &&
-					createParam.sequenceNumber <= parentLatestSummary.referenceSequenceNumber
+					this._lastSummaryReferenceSequenceNumber !== undefined &&
```

Comment:
nit: I suggest using either `parentLastSummaryReferenceSequenceNumber` throughout this function of not creating that variable in the first place. Right now, it's using a combination of `parentLastSummaryReferenceSequenceNumber` and `this._lastSummaryReferenceSequenceNumber` which seems weird.